staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -642,6 +672,11 @@ type PersistentVolumeClaimStatus struct {
 	// Represents the actual resources of the underlying volume.
 	// +optional
 	Capacity ResourceList `json:"capacity,omitempty" protobuf:"bytes,3,rep,name=capacity,casttype=ResourceList,castkey=ResourceName"`
+	// Current service state of PVC
```

Comment:
Please add more description here about what conditions are normally set and how an end user might interpret this.

Reply:
done